third_party/move/move-vm/runtime/src/storage/implementations/unsync_module_storage.rs

```diff
@@ -177,12 +188,12 @@ impl<'s, S: ModuleBytesStorage, E: WithRuntimeEnvironment> UnsyncModuleStorage<'
         assert_eq!(self.0.num_modules(), deserialized.len() + verified.len());
         for id in deserialized {
             let result = self.0.get_module_or_build_with(id, &self.0);
-            let module = claims::assert_some!(claims::assert_ok!(result));
+            let module = claims::assert_some!(claims::assert_ok!(result)).0;
```

Comment:
anything to check for the version?

Reply:
For unsync, we always use `NoVersion` here so no point of checking.